src/service_daemon.rs

```diff
@@ -2356,6 +2356,21 @@ impl DnsCache {
             _ => return None,
         };
 
+        if incoming.get_cache_flush() {
+            // Mark all existing records of this type as expired, and prepend the new record.
+            record_vec.iter_mut().for_each(|r| {
+                // When cache flush is asked, we set expire date to 1 second in the future
+                // if created more than 1 second ago
+                // Ref: RFC 6762 Section 10.2
+                let now = current_time_millis();
```

Comment:
I think it's better to only call `current_time_millis` once before entering `for_each` loop. The time accuracy loss should near 0 (for millisecond)

Reply:
You're right. Moved it.